packages/eslint-plugin/tests/RuleTester.ts

```diff
@@ -22,7 +22,7 @@ interface InvalidTestCase<
   TOptions extends Readonly<any[]>
 > extends ValidTestCase<TOptions> {
   errors: TestCaseError<TMessageIds>[];
-  output?: string;
+  output?: string | null;
```

Comment:
Not a blocker, just checking we need `string`, `null` and `undefined` as possible values here?

Reply:
We can use three patterns of `output`.

- no `output` property ... This means the rule doesn't support autofix at all. The tester does nothing about autofix.
- `output: string` ... This means the rule fixes code automatically in the test case. The tester verifies that code was fixed correctly.
- `output: null` ... This means the rule doesn't fix code in the test case. The tester verifies the rule didn't fix code.